### PR TITLE
Add offline events to Philips Hue

### PR DESCRIPTION
**Pull request by larsfinander**
-Handles case when bridge goes offline
-Handles case when indiviudual lights go offline

### CODE REVIEW

**Comment by larsfinander** [comment]
@workingmonk @dchem PLease review, using the new offline events for device watch

**Comment by larsfinander** [comment]
@jackchi Can you review?

**Comment by dchem** [comment]
:+1: lgtm

File: smartapps/smartthings/hue-connect.src/hue-connect.groovy [hunk 1]
```diff
@@ -740,16 +740,21 @@ private void checkBridgeStatus() {
...
-	        } else {
+			if (it.value.lastActivity < time) { // it.value.lastActivity != null &&
+				log.warn "Bridge $it.key is Offline"
+				d.sendEvent(name: "status", value: "Offline")
+
+				state.bulbs?.each {
+					it.value.online = false
+				}
+				getChildDevices().each {
+					it.sendEvent(name: "DeviceWatch-DeviceOffline", value: "offline")
```
Comment (jackchi) [inline smartapps/smartthings/hue-connect.src/hue-connect.groovy:751]: Do we need `isStateChange:true` or `displayed:false` in these events?

**Pull request merged**